src/NzbDrone.Core/Download/TrackedDownloads/TrackedDownloadAlreadyImportService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.History;
+
+namespace NzbDrone.Core.Download.TrackedDownloads
+{
+    public interface ITrackedDownloadAlreadyImportService
```

Comment:
Don't think this is really a 'service'. ITrackedDownloadAlreadyImported should do it right?